doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -141,3 +141,11 @@ By default this Kconfig option is set to 1 second.
 .. note::
   The :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is set to ``y`` by default.
   To disable removing application-specific non-volatile data when the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT` Kconfig option is selected, set the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option to ``n``.
+
+Read Client functionality
+=========================
+
+Read Client functionality needs to be enabled if the device needs to read attributes from another device in the Matter network.
```

Comment:
What about: The **Read Client** feature shall be activated when the device requires the capability to access attributes from a different device within the Matter network.